### ISSUE TITLE
Issue with TURN Server

### ISSUE THREAD

**Issue by mjsobrep** [Rehab-Robotics-Lab/LilFloSystem#231]
The turn server isn't working. Uh oh

The ICE Server Provider isn't getting a username and password back from the web server:

```
Node: /ice_server_provider
Time: 15:35:19.619415044 (2021-06-22)
Severity: Warn
Published Topics: /rosout

response did not have username and password fields

Location:
ice_server_service.py:IceServerManager.get_turn_creds:45

----------------------------------------------------------------------------------------------------

```

**Comment by mjsobrep** [comment]
This comes out of the ice server provider here: https://github.com/RobotWebTools/webrtc_ros/blob/ebd3638239682138a47d620a60fce28131e59aad/webrtc_ros/scripts/ice_server_service.py#L45

**Comment by mjsobrep** [comment]
password and username come from here:

https://github.com/Rehab-Robotics-Lab/LilFloSystem/blob/3b5ee49273f09fef8cb4d82cd47aa4a84223dbe6/flo_telepresence/launch/webrtc.launch#L26-L28

**Comment by mjsobrep** [comment]
The error on the server side comes from here:
https://github.com/Rehab-Robotics-Lab/LilFloSystem/blob/3b5ee49273f09fef8cb4d82cd47aa4a84223dbe6/flo_web/web_server/src/routes/webrtc.ts#L32-L86

Where the error just shows up as: 
error: true

 a|b
-- | --
method | POST
route| /api/webrtc/turn-credentials
status_code|500
url|http://lilflo.com/api/webrtc/turn-credentials 
method | POST 

So I think it is coming from the last line here. Might want to put a bit more info in that error message to check for sure

**Comment by mjsobrep** [comment]
closed by #242
